ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```diff
@@ -15,12 +33,16 @@ tests:
 - as: unit-tests
   commands: ci/prow/unit-tests
   container:
-    from: src
+    clone: true
+    from: ocp_builder_rhel-8-golang-1.18-openshift-4.12
```

Comment:
Is this the most basic image that we get in CI?

Reply:
That's one that has Go 1.18 and `make`, at least. Happy to switch if you find anything smaller.

Reply:
That is OK. I didn't dig for all the images but it seems decent to me.